libs/wallet/src/api/hooks/useWalletCapabilities.ts

```diff
@@ -61,9 +61,7 @@ export function useWalletCapabilities(): SWRResponse<WalletCapabilities | undefi
               return
             }
 
-            const chainIdHex = '0x' + (+chainId).toString(16)
-
-            resolve(result[chainIdHex])
+            resolve(result[Object.keys(result)[0]])
```

Comment:
Hmmm, weird.
Corresponding to the [specification](https://eips.ethereum.org/EIPS/eip-5792#wallet_getcapabilities), it should be:

<img width="581" height="549" alt="image" src="https://github.com/user-attachments/assets/779b999d-9055-484d-9cc0-58176906afd0" />